### PR TITLE
add favorite list 

### PR DESCRIPTION
**Pull request by orange-na**

### CODE REVIEW

File: js/handle_storage.js [hunk 1]
```diff
@@ -36,13 +49,25 @@ const favoriteBtn = () => {
...
+        );
+        if (optionToDelete) {
+            favoriteList.removeChild(optionToDelete);
+        }
     } else {
         $iconFill.style.display = "block";
         favoriteCities.push(newFavoritecity);
         localStorage.setItem("favorite", JSON.stringify(favoriteCities));
-        alert("The city has saved successfully!!");
+        console.log(newFavoritecity);
```
Comment (obrkn) [inline js/handle_storage.js:63]: delete please

**Review by obrkn — COMMENTED** [review]
There are the same city options. If I changed the city and favorite, it won't be stocked in localStorage.
 
Screen Recording 2023-08-09 at 10 47 09 AM (1) (1)

**Comment by orange-na** [comment]
I fixed these bugs.

**Review by obrkn — APPROVED** [review]
LGTM

**Pull request merged**